README.md

```diff
@@ -127,9 +127,9 @@ or
 
 ### License
 
-The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [The 3-Clause BSD License].
+The source code TIA Toolbox (tiatoolbox) as hosted on GitHub is released under the [BSD-3-Clause license](https://github.com/TissueImageAnalytics/tiatoolbox/blob/develop/LICENSE). The full text of the licence is included in [LICENSE](https://raw.githubusercontent.com/TissueImageAnalytics/tiatoolbox/develop/LICENSE).
 
-The full text of the licence is included in [LICENSE](https://raw.githubusercontent.com/TissueImageAnalytics/tiatoolbox/develop/LICENSE).
+Models weights are dependent on the datasets that they were trained on. Refer to the [documentation](https://tia-toolbox.readthedocs.io/en/latest/index.html) for more details.
 
```

Comment:
- [ ] Please also add this link https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE to https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE

- [ ] Please change line 25 to <a href="#license">

Reply:
The first comment I don't understand - both links are the same.

Reply:
sorry, please add this link https://tia-toolbox.readthedocs.io/en/latest/pretrained.html to https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE. To clarify that model weights are under a different license.

You can change the https://github.com/TissueImageAnalytics/tiatoolbox/blob/doc-update-model-licenses/LICENSE file by adding this to the top.
```
BSD 3-Clause License

Model weights are provided under a different license. For details, please see https://tia-toolbox.readthedocs.io/en/latest/pretrained.html

Copyright (c) 2022, Tissue Image Analytics (TIA) Centre
All rights reserved.
```